app/screens/login/login.js

```diff
@@ -205,7 +194,13 @@ export default class Login extends PureComponent {
     signIn = () => {
         const {actions, loginId, loginRequest, password} = this.props;
         if (loginRequest.status !== RequestStatus.STARTED) {
-            actions.login(loginId.toLowerCase(), password);
+            actions.login(loginId.toLowerCase(), password).then(this.checkLoginResponse);
+        }
+    };
+
+    checkLoginResponse = (data) => {
+        if (data?.error.server_error_id === 'mfa.validate_token.authenticate.app_error') { // eslint-disable-line camelcase
```

Comment:
That's really weird that this would cause ESLint to complain when it's supposed to allow this for property access. Maybe it's some conflict with the transpiler for `?.`

Reply:
Was really surprised about that as well, I can move the undefined check to a separate condition or keep the ignore comment, personally I don't mind.